blueocean-core-js/package.json

```diff
@@ -5,8 +5,7 @@
   "main": "dist/js/index.js",
   "scripts": {
     "gulp": "gulp",
-    "test": "gulp test",
-    "prepublish": "gulp test; gulp"
```

Comment:
I removed prepublish because anytime you do `npm install` it runs this, and if you are sorting out shrink issues this tends to fail all the time.